tests/integration/test_httpx.py

```diff
@@ -275,42 +273,19 @@ def response_cookies(response):
             assert response_cookies(cassette_response) == ["k1", "k2"]
             assert client_cookies(new_client) == ["k1", "k2"]
 
-
 @pytest.mark.online
-def test_relative_redirects(tmpdir, scheme, do_request, mockbin):
-    redirect_kwargs = {HTTPX_REDIRECT_PARAM.name: True}
+def test_stream(tmpdir, httpbin, do_request):
+    url = httpbin + "/stream-bytes/512"
+    testfile = str(tmpdir.join("stream.yml"))
 
-    url = mockbin + "/redirect/301?to=/redirect/301?to=/request"
-    testfile = str(tmpdir.join("relative_redirects.yml"))
     with vcr.use_cassette(testfile):
-        response = do_request()("GET", url, **redirect_kwargs)
-        assert len(response.history) == 2, response
-        assert response.json()["url"].endswith("request")
+        response_content = do_request().stream("GET", url)
+        assert len(response_content) == 512
+        
 
     with vcr.use_cassette(testfile) as cassette:
-        response = do_request()("GET", url, **redirect_kwargs)
-        assert len(response.history) == 2
-        assert response.json()["url"].endswith("request")
-
-        assert cassette.play_count == 3
-
-
-@pytest.mark.online
-def test_redirect_wo_allow_redirects(do_request, mockbin, yml):
```

Comment:
I think it would be cool to keep the tests with follow redirects param with true and false